# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/src/WRHitTester.cpp
# Commit: 7d85a597c9ae
# Full Hash: 7d85a597c9aec41cbde40d01172539cf90351156
# Author: Dan Robertson <drobertson@mozilla.com>
# Date: 2022-09-24 09:33:46
# Regressor Bug: 1778120
# File Overlap Count: 1
# Description:
#   Bug 1778120 - Add the animation id to the webrender stacking context. r=botond
#   
#   On desktop add the animation id to the webrender stacking context.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D158035
# ==============================================================================

diff -r 9695f66aa0fe -r 7d85a597c9ae gfx/layers/apz/src/WRHitTester.cpp
--- a/gfx/layers/apz/src/WRHitTester.cpp	Fri Sep 23 20:16:16 2022 +0000
+++ b/gfx/layers/apz/src/WRHitTester.cpp	Fri Sep 23 20:44:02 2022 +0000
@@ -221,7 +221,6 @@
                                            positionedNode);
     }
 
-#if defined(MOZ_WIDGET_ANDROID)
     if (hit.mNode && hit.mNode->GetFixedPositionAnimationId().isSome()) {
       // If the hit element is a fixed position element, the side bits from
       // the hit-result item tag are used. For now just ensure that these
@@ -234,7 +233,6 @@
       MOZ_ASSERT(sideBits == SideBits::eNone,
                  "Hit test results have side bits only for pos:fixed");
     }
-#endif
   }
 
   hit.mHitOverscrollGutter =